### PR TITLE
Update vs_code_welcome_page.ts

### LINKED ISSUE #35719: [BUG] Activity Bar refresh bug over loading extension marketplace list
- VSCode Version: 1.17.0 c[ be377c0faf7574a59f84940f593a6849f12e4de7 ]
- OS Version: Windows 8.1 64-bit & Linux Ubuntu 16.04.2 Gnome 3

Steps to Reproduce:

---

[Lang = "CZ/SK"]
Ak budete veľmi rýchlo skrývať a zobrazovať ikony v activity bar, tak bežne sa v panelu zobrazuje práve ten ktorý bol naposledy odkrytý alebo ten ktorý je nad skrytým, to je ok, ale ak takýmto spôsobom otvoríte extension marketplace list, a zároveň počas načitávania extensions bude naďaľej rýchlo skrývať a zobrazovať ikony v Activiry Bar, tak občas sa stane že sa naďalej graficky bude ukazovať ikona v activity bar, niekedy funguje na ňu kliknúť, niekedy nie, ovšem v zoznamu "show/hide" nieje pri danej ikone/položke fajka. Alebo naopak je, ale ikona / položka sa nezobrazuje, niekedy ovšem keď kliknete na prázdne miesto v activity bar, otvoríte aj skrytú položku. Problém je s reprodukovaním, pretože sa to udeje náhodne a len občas / zriedkavo. Ak sa to stane, tak celý panel a aktivity bar je zabugovaný až kým nevypnete a znova nezapnete vscode.

---

[EN - translated by google translator]

If you hide and display icons in the activity bar very quickly, so the one that was last exposed or the one that is hidden is usually in the panel, but if you open the extension marketplace list, you will also be able to open the extension extensions to quickly hide and display icons in the Activiry Bar, sometimes it happens that the icon in the activity bar will continue to be shown graphically, sometimes it's clickable on it, sometimes not, but in the "show / hide" list it is not the icon / brier item. Or vice versa, but the icon / item is not displayed, but sometimes when you click on a blank space in the activity bar, you also open the hidden item. The problem is with reproduction because it happens at random and only occasionally / rarely. If this happens, the entire panel and activity bar is bugged, you must close and again open vscode to fix it.

---

Reproduces without extensions: Yes

screenshot_84

---
[lang = "CZ/SK"]
P.s. Ak niekto vie prekladať z češtiny alebo slovenčiny do angličtiny, prosím o pomoc, totiž neco málo po anglicky viem, a neprijde mi že preklad je správny. (aj keď si to dám preložiť spätne, tak to preloží chujovinu).

[EN - translated by google translator]
P.s. If someone can translate from Czech or Slovak into English, please help, I know little about English and I do not think the translation is correct. (even though I'm going to translate it back, so it translates the stupid

### LINKED ISSUE #35764: [BUG] Recent in Welcome page is not the same as Recent in Task Bar
- VSCode Version: 1.17.0 c[ be377c0faf7574a59f84940f593a6849f12e4de7 ]
- OS Version: Windows 8.1 64-bit & Linux Ubuntu 16.04.2 Gnome 3

Steps to Reproduce:

1. Clear Recent list, and kill vscode, again open vscode

Reproduces without extensions: Yes

screenshot_94

### PR DESCRIPTION
**Pull request by ghost**
FIX #35764 by add refresh if Welcome page if is opened, and you clear "recent list".

Not already solved.

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
https://github.com/Mlocik97-issues/vscode/tree/patch-1


(First commit in PR was mistake)

File: src/vs/workbench/parts/welcome/page/electron-browser/vs_code_welcome_page.ts [hunk 1]
```diff
@@ -77,4 +77,4 @@ export default () => `
 	</div>
 </div>
 `;
-// #35719
+// #35764
```
Comment (Mlocik97-issues) [inline src/vs/workbench/parts/welcome/page/electron-browser/vs_code_welcome_page.ts]:
```
<div class="section recent" id="recentL">
					<h2 class="caption">${escape(localize('welcomePage.recent', "Recent"))}</h2>
					<ul class="list">
						<!-- Filled programmatically -->
						<li class="moreRecent"><a href="command:workbench.action.openRecent">${escape(localize('welcomePage.moreRecent', "More..."))}</a><span class="path detail if_shortcut" data-command="workbench.action.openRecent">(<span class="shortcut" data-command="workbench.action.openRecent"></span>)</span></li>
					</ul>
					<p class="none detail">${escape(localize('welcomePage.noRecentFolders', "No recent folders"))}</p>
                                       /* 
                                        if () {
					      parent.document.getElementById("recentL").reload();
                                        } 
*/
				</div>
```

**Comment by chrmarti** [comment]
@Mlocik97-issues The change seems to be missing, I see only a single line change: https://github.com/Microsoft/vscode/pull/38096/files

**Comment by chrmarti** [comment]
Please resubmit, the change somehow didn't make to this PR.

**Pull request closed without merging**